unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/NavmapZoomView.cs

```diff
@@ -18,6 +18,24 @@ public class NavmapZoomView : MonoBehaviour
         internal ZoomInput ZoomOut => zoomOut;
         internal InputAction_Measurable MouseWheelAction => mouseWheelAction;
 
+        private void Awake()
+        {
+            if (DataStore.i.featureFlags.flags.Get().IsInitialized)
```

Comment:
would be better to inject the DataStore from the controller as we do usually with an Initialize method or something like that